src/languages/en.ts

```diff
@@ -2450,6 +2456,17 @@ export default {
             existingConnections: 'Existing connections',
             sageIntacctLastSync: (formattedDate: string) => `Sage Intacct - Last synced ${formattedDate}`,
         },
+        sageIntacct: {
+            autoSync: 'Auto-sync',
+            autoSyncDescription: 'Sync Sage Intacct and Expensify automatically, every day.',
+            inviteEmployees: 'Invite employees',
+            inviteEmployeesDescription:
+                'Import Sage Intacct employee records and invite employees to this workspace. Your approval workflow will default to manager approval and can be furthered configured on the Members page.',
```

Comment:
NAB: does this need to be on a new line?

Reply:
Unfortunately, yes. Prettier does that 🤷